dissect/target/plugins/os/unix/packagemanager.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+from enum import Enum
+from typing import Iterator
+
+from dissect.target import Target
+from dissect.target.exceptions import UnsupportedPluginError
+from dissect.target.helpers.record import TargetRecordDescriptor
+from dissect.target.plugin import Plugin, export
+
+PackageManagerLogRecord = TargetRecordDescriptor(
+    "linux/log/packagemanager",
+    [
+        ("datetime", "ts"),
+        ("string", "package_manager"),
+        ("string", "operation"),
+        ("string", "package_name"),
+        ("string", "command"),
+        ("string", "requested_by_user"),
+    ],
+)
+
+
+class OperationTypes(Enum):
+    Install = "install"
+    Update = "update"
+    Downgrade = "downgrade"
+    Remove = "remove"
+    Other = "other"
+
+    __MAPPING__: dict = {
+        Install: ["install", "installed", "reinstall"],
+        Update: ["update", "updated", "upgrade"],
+        Downgrade: ["downgrade"],
+        Remove: ["remove", "removed", "erased", "purge"],
+        Other: ["command"],
+    }
+
+    @classmethod
+    def infer(cls, keyword: str) -> OperationTypes:
+        keyword = keyword.strip().lower()
+        for key, values in cls.__MAPPING__.items():
+            if keyword in values:
+                return OperationTypes(key)
+        return OperationTypes.Other
+
+
+class PackageManagerPlugin(Plugin):
+    __namespace__ = "packagemanager"
+
```

Comment:
```suggestion
    __findable__ = False
```

Feature in new CLI interface that allows to skip this if used together with wildcard: packagemanager.*